codegen/src/api/mod.rs

```diff
@@ -211,35 +271,36 @@ impl RuntimeGenerator {
 
         let metadata_hash = get_metadata_per_pallet_hash(&self.metadata, &pallet_names);
 
-        let modules = pallets_with_mod_names.iter().map(|(pallet, mod_name)| {
+        let mut modules = Vec::new();
```

Comment:
I don't think you need to use a `Vec` here as as quote is implemented for iterators as well but it's fine either way